packages/ark/source/confirmed-transaction.dto.ts

```diff
@@ -63,7 +63,16 @@ export class ConfirmedTransactionData extends DTO.AbstractConfirmedTransactionDa
 		}
 
 		if (this.isMultiPayment()) {
-			return this.recipients().some(({ address }: Contracts.MultiPaymentRecipient) => address === this.sender());
+			let isReturn = true;
+
+			for (const recipient of this.recipients().values()) {
+				if (recipient.address !== this.sender()) {
+					isReturn = false;
+					break;
+				}
+			}
+
+			return isReturn;
```

Comment:
This change ensures that `isReturn` will return `true` only when all of the recipients are the same as sender.